### ISSUE TITLE
Decide on a database vendor

### ISSUE THREAD

**Issue by RichardInnocent** [adlkucewicz/ChatIO#6]

**Comment by RichardInnocent** [comment]
I don't think we need anything particularly advanced. As this is a personal project, I'd rather have something that's relatively easy to manage on my laptop. Maybe MySQL? We can always change this later if we want to.

**Comment by adlkucewicz** [comment]
SQLite is really easy to use with django. In fact i'm pretty sure there is already a file for it in chatio_django.

**Comment by RichardInnocent** [comment]
> SQLite is really easy to use with django. In fact I'm pretty sure there is already a file for it in chatio_django.

SQLLite isn't particularly good for faster or distributed models but I'm happy to stick with it at this point. We're probably not building something that other people will use so I'm fine with that.

We may have to switch later if we think about deploying this somewhere - we'll see.
